mnemosyne/cli.py

```
@@ -699,6 +720,43 @@ def _progress(store, done, total):
+    bank_override = None
+    i = 0
+    while i < len(args):
+        if args[i] == "--bank" and i + 1 < len(args):
+            bank_override = args[i + 1]
+            i += 2
+        else:
+            i += 1
+    if bank_override:
+        bank = bank_override
+    else:
+        bank = os.environ.get("MNEMOSYNE_BANK", "").strip() or "default"
```

Ruff (0.15.20)

[warning] Use binary operator `bank = bank_override or (os.environ.get("MNEMOSYNE_BANK", "").strip() or "default")` instead of `if`-`else`-block

Replace `if`-`else`-block with `bank = bank_override or (os.environ.get("MNEMOSYNE_BANK", "").strip() or "default")`

(SIM108)